Dockerfile

```diff
@@ -23,16 +23,15 @@
 # the case. Therefore, you don't have to disable it anymore.
 #
 
-FROM ubuntu:14.04
-MAINTAINER Tianon Gravi <admwiggin@gmail.com> (@tianon)
```

Comment:
Heh, removed because I don't see what value the instruction adds besides an
extra layer and an extra point where build cache might bust -- can add it
back if there's a good reason, but I haven't seen tooling take good
advantage of the metadata it provides, so I can't personally justify using
it.